src/styles/general.scss

```diff
@@ -64,3 +71,7 @@
 		transform: scale(1.2);
 	}
 }
+
+@font-face {
+	font-family: 'Roboto', sans-serif;
+}
```

Comment:
This should be moved into an extra `style` tag in the `app.html`, because otherwise it would cause build errors on startup. 
This could be fixed by changing the sass preprocessing configuration for svelte, but is too complicated in this case. 
So just move these lines into the `app.html` file

Reply:
second part of the fix is in a second pull request :)